build-utils/css/utils/convertToCssVariable.ts

```diff
@@ -0,0 +1,8 @@
+export const convertToCssConstant = (themeContent: Record<string, string>) => {
+  const newContent = {};
+  Object.entries(themeContent).map(([key, value]) => {
+    newContent[`--amino-${key}`] = value;
+    return null;
+  });
```

Comment:
The return null is a little awkward here. I'm guessing it satisfied a lint rule? What do you think about using a `forEach` instead?
```suggestion
  Object.entries(themeContent).forEach(([key, value]) => {
    newContent[`--amino-${key}`] = value;
  });
```